setup.py

```diff
@@ -69,9 +69,9 @@ def get_requirements(requirements_filename: str):
         "Programming Language :: Python :: 3",
         "Operating System :: OS Independent"
     ],
-    python_requires='>=3.6',
+    python_requires='>=3.7',
     install_requires=get_requirements("requirements.txt"),
-    extras_require={"gradio": get_requirements("gradio.txt")},
+    extras_require={"gradio": get_requirements("gradio.txt"), "web_sat": get_requirements("web_sat.txt")},
```

Comment:
Probably need to add some package data spec to include the `static` and `templates` directories